### ISSUE TITLE
ResNet50 example should not have biases for Conv modules

### ISSUE THREAD

**Issue by tomhennigan** [google/jax#3203]
The [ResNet50 example](https://github.com/google/jax/blob/master/examples/resnet50.py) has too many parameters (161 are expected but stax produces 214). This is afaik because the conv layers include biases where they should not. Per the paper author [0] the batchnorm layers should contain the biases.

The delta comes from (1x) the initial conv (4x 4) from the conv blocks and (12x 3) from the identity blocks (leading to the extra 53 parameters).

[0] https://github.com/KaimingHe/deep-residual-networks/issues/10

**Comment by tomhennigan** [comment]
BTW I think folks are also very likely to fork this code given its prominent location in the examples dir, I think we should be clearer about:

1. How confident are we in the implementation, has it been shown to reproduce any published experiment?
2. Is this ResNetV1 or V2?
3. Has this implementation reproduced any published experiments?
4. What statistics are used during test time (a: "local batch, same as training").

**Comment by hawkinsp** [comment]
We removed this example long ago!
